.github/workflows/linkcheck.yml

```
@@ -12,6 +12,8 @@ on:
     branches:
     - main
 
+permissions: {}
```

```suggestion
permissions: {}  # disables all GitHub permissions for the workflow
```
Same as [comment](https://github.com/PlasmaPy/PlasmaPy/pull/2959/files#r2055075406)